docs/architecture/adr-template.md

```diff
@@ -0,0 +1,61 @@
+# ADR {ADR-NUMBER}: {TITLE}
+
+## Changelog
+
+- {date}: {changelog}
+
+## Status
+
+{DRAFT | PROPOSED} Not Implemented
+
+> Please have a look at the [PROCESS](./PROCESS.md#adr-status) page.
+> Use DRAFT if the ADR is in a draft stage (draft PR) or PROPOSED if it's in review.
```

Comment:
```suggestion
> For details on ADR workflow, see the [PROCESS](./PROCESS.md#adr-status) page.
> Use DRAFT if the ADR is in a draft stage (draft PR) or PROPOSED if it's in review.
```

Reply:
we typically avoid using please, see https://developers.google.com/style/tone#politeness-and-use-of-please